extensions/interactions/NumericInput/protractor.js

```diff
@@ -32,6 +32,8 @@ var expectInteractionDetailsToMatch = function(elem) {
 var submitAnswer = function(elem, answer) {
   elem.element(by.tagName('oppia-interactive-numeric-input')).
     element(by.tagName('input')).sendKeys(answer + '\n');
```

Comment:
I remember something like that, but do you remember why exactly it didn't work when there was a form in there?